main.tf

```diff
@@ -5,14 +5,21 @@ resource "local_file" "this" {
   filename = "${trimsuffix(var.target_dir, "/")}/${var.name}/${each.value.name}.tf"
 }
 
+resource "tfe_project" "project" {
+  count = var.workspace.project != "" ? 1 : 0
```

Comment:
what about case when you need to link(use existing project which have been created with another module) with existing project. have we data resource for tfe_project so that this will work?
```hcl
data "tfe_project" "project" {
  count = var.workspace.project != "" ? 1 : 0
  ....
}
````
and in places you use this will be
```hcl
project_id = try(tfe_project.project[0].id, data.tfe_project.project[0].id)
```